orcestra/postprocess/level0.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+
+def _bahamas_fix_time(ds):
+    """Fix time coordinate of BAHAMAS datasets."""
+    return ds.rename({"tid": "time", "TIME": "time"}).set_index(time="time")
+
+
+def _bahamas_line_of_sight(ds):
+    """Calculate the length of line-of-sight (LOS)."""
+    return ds.assign(
+        LOS=lambda dx: dx.IRS_ALT
+        / np.cos(np.radians(dx.IRS_THE))
+        / np.cos(np.radians(dx.IRS_PHI))
+    )
```

Comment:
The `LOS` variable seems to assume a certain sensor viewing direction (e.g. the radar), thus is not really a variable which should generically be associated with the basic instrumentation system. Instead, this variable likely should be attached to the instrument it's referring to.

Reply:
I am fine with dropping this processing step